Multiple ajout (voir description)

QuentinGrnr opened the pull request:
Encore désolé d'avoir fait autant de modifs dans un seul comit, la prochaines fois j'en ferais plusieurs. J'ai aussi retiré les balises html.

> - Ajout du KOTH / Totem / Chest refill et de leur planning
> - Nouvelle catégorie: ``boss``, ``eventsauto``, ``metiers``
> - Retrait de tag nouveau, plus d'actualité

QuentinGrnr committed:
rebase

QuentinGrnr committed:
Modification équilibrage métier

ShockedPlot7560 committed:
Merge branch 'stable' into stable

QuentinGrnr committed:
Merge pull request #1 from Plutonium-Mcpe/stable

ajout totem, chest, koth

QuentinGrnr committed:
Chest refield + remove and set NEW

QuentinGrnr committed:
Chest refield + remove and set NEW

QuentinGrnr committed:
Add doc KOTH + Chest + totem + planning / creation folder boss + eventauto + metiers (réorganisation)

Encore désolé d'avoir fait autant de modifs dans un seul comit, la prochaines fois j'en ferais plusieurs. J'ai aussi retiré les balises html.

QuentinGrnr committed:
Add doc KOTH + Chest + totem + planning / creation folder boss + eventauto + metiers (réorganisation)

Encore désolé d'avoir fait autant de modifs dans un seul comit, la prochaines fois j'en ferais plusieurs. J'ai aussi retiré les balises html.

ShockedPlot7560 commented on docs/boss/boss_category.md:
```diff
@@ -0,0 +1,7 @@
+id: boss
+title: Les boss
+description: Sortez votre épée, les boss vous attend !
+icon: " "
```
Il est préferable de mettre un icône à la catégorie, sinon la page d'accueil du wiki n'en contiendras pas.  
Peut-être un des boss, ou quelque chose qui fait combat ?

ShockedPlot7560 commented on docs/eventsauto/chestrefield.md:
```diff
@@ -0,0 +1,40 @@
+---
+id: chestrefield
```
Plutôt appeler le fichier et l'id: ``chest-refill`` ?

ShockedPlot7560 commented on docs/eventsauto/chestrefield.md:
```diff
@@ -0,0 +1,40 @@
+___
+## Pourquoi des coffres en Warzones ?
+En vous baladant dans les warzones de chaque serveur, vous pouvez tomber sur des coffres avec du stuff.
+Toute les **4 heures** un des **6 coffres** répartis sur les 3 serveurs se rempli de stuff.
```
``se remplissent de stuff``

ShockedPlot7560 commented on docs/eventsauto/chestrefield.md:
```diff
@@ -0,0 +1,40 @@
+Soyez le premier sur les lieux pour récupérer ce stuff !
+
+## Quelles sont les récompenses possibles de ses derniers ?
+Le tableau ci-dessous vous permet de prendre connaissance des probabilities d'apparition de chaque loot :
```
``probabilitées``

ShockedPlot7560 commented on docs/eventsauto/chestrefield.md:
```diff
@@ -0,0 +1,40 @@
+
+## Où trouver ces coffres
+### Serveur Stelite :
+**Coffre N°1** : -325 91 -339  
```
Mettre pour les coordonnées, pour éviter toute ambiguïté, à quel axe correspont chaque chiffres ?

ShockedPlot7560 commented on docs/eventsauto/chestrefield.md:
```diff
@@ -0,0 +1,40 @@
+**Coffre N°2** : -184 89 184
+
+#### Partez à leur recherche ! Mais attention, ces derniers sont très convoités et vous pouvez vous retrouver face a d'autres joueurs !
+<img style="margin: 0 auto;" src="https://user-images.githubusercontent.com/109299545/182045362-5120af39-74e0-4ae2-9392-105f9a2e7280.PNG">
```
L'image, en la recadrant en format 1:1 pourrait être mises en tant que thumbnail en haut de la page.  Ca pourrait illustrer un peu mieux
> Ajouter la classe ``thumbnail-right`` à la balise HTML

ShockedPlot7560 commented on docs/eventsauto/chestrefield.md:
```diff
@@ -0,0 +1,40 @@
+**Coffre N°1** : 199 90 -196  
+**Coffre N°2** : -184 89 184
+
+#### Partez à leur recherche ! Mais attention, ces derniers sont très convoités et vous pouvez vous retrouver face a d'autres joueurs !
```
Plutôt que de le mettre en titre, je l'aurais plutôt vu en temps que texte final normal, sans le format de titre

ShockedPlot7560 commented on docs/eventsauto/eventsauto_category.md:
```diff
@@ -0,0 +1,7 @@
+---
+id: eventsauto
+title: Events automatiques
```
Plutôt qu'event, j'aurais mit événements

ShockedPlot7560 commented on docs/eventsauto/eventsauto_category.md:
```diff
@@ -0,0 +1,7 @@
+id: eventsauto
+title: Events automatiques
+description: Besoins de renseignement concernant les events automatiques ?
+icon: " "
```
Il y auras besoin d'une image, une horloge ?

ShockedPlot7560 commented on docs/eventsauto/koth.md:
```diff
@@ -0,0 +1,31 @@
+title: [NEW] Les koth en warzone
+category: eventsauto
+description: Restez le plus longtemps dans la zone !
+icon: "https://user-images.githubusercontent.com/109299545/182051667-27abd4dd-66a8-412b-bb39-c09cf4c0ad8a.png"
```
Préfère l'utilisation des images venant du texture pack vanilla, ou de pluto.

ShockedPlot7560 commented on docs/eventsauto/koth.md:
```diff
@@ -0,0 +1,31 @@
+### Etape 1 : Se rendre aux coordonnées de la zone de combat avec sa faction
+Les heures durant lesquelles se déroulent l'évent KOTH sont indiquée dans #eventauto/planning
+
+**Stelite** : 201 80 -292  
```
Mettre à quel axe correspond chaque chiffre ?

ShockedPlot7560 commented on docs/eventsauto/koth.md:
```diff
@@ -0,0 +1,31 @@
+**Stelite** : 201 80 -292  
+**Arcadia** : -184 85 -216 et 201 80 -292
+
+<img style="margin: 0 auto;" src="https://user-images.githubusercontent.com/109299545/182142753-38d1a60f-19fb-48b8-b52d-70b90b2d20ca.png" alt="le KOTH du stelite">
```
Reformater l'image en 1:1 et la placer en thumbnail en haut de la page ?
> Utiliser la classe ``thumbnail-right`` sur la balise
> *Voir l'article des boss pour plus de détails*

ShockedPlot7560 commented on docs/eventsauto/planning.md:
```diff
@@ -0,0 +1,12 @@
+---
+id: planning
+title: [NEW] Planning des events automatique 
```
des ``événements``

ShockedPlot7560 commented on docs/eventsauto/planning.md:
```diff
@@ -0,0 +1,12 @@
+---
+___
+
+Voici un agenda relatant les heures et jours durant lesquels les event automatiques se lancent.
```
Plutôt que relatant, j'aurais mit récapitulant, et mêmes choses pour event

ShockedPlot7560 commented on docs/eventsauto/totem.md:
```diff
@@ -0,0 +1,31 @@
+title: [NEW] Les totems en warzone
+category: eventsauto
+description: Soyez les premier a détruire le totem 
+icon: "https://user-images.githubusercontent.com/109299545/182046178-97bf8b14-9f27-4dc3-a8bf-0556e2cddc21.png"
```
Utiliser les images du textures pack de préférences.  
Choses à part, j'aurais soit choisi ce totem, soit le bloc du totem directement ?

ShockedPlot7560 commented on docs/eventsauto/totem.md:
```diff
@@ -0,0 +1,31 @@
+### Etape 1 : Se rendre aux coordonnées avec sa faction
+Les heures durant lesquelles se déroulent l'évent totem sont indiquée dans #eventauto/planning  
+
+**Stelite** : -265 74 268  
```
Préciser à quel axe correspond chaque chiffres?

ShockedPlot7560 commented on docs/eventsauto/totem.md:
```diff
@@ -0,0 +1,31 @@
+**Stelite** : -265 74 268  
+**Arcadia** : 258 89 278
+
+<img style="margin: 0 auto;" src="https://user-images.githubusercontent.com/109299545/182133858-054a2d53-7bf6-47cd-82cf-12ab02ea6a51.png" alt="le totem du stelite">
```
Reformater l'image en 1:1 et la placer en thumbnail en haut de la page ?
> Utiliser la classe ``thumbnail-right`` sur la balise
> *Voir l'article des boss pour plus de détails*

ShockedPlot7560 commented on docs/metiers/metiers_category.md:
```diff
@@ -0,0 +1,7 @@
+id: metiers
+title: Les metiers
+description: Tout sur les metiers !
+icon: " "
```
Un icône pour la catégorie est préférable

ShockedPlot7560 commented on docs/metiers/metiers_category.md:
```diff
@@ -0,0 +1,7 @@
+---
+id: metiers
```
j'aurais mit ``jobs`` pour l'id de l'article qui sont en anglais majoritairement, contrairement au contenu de l'article

ShockedPlot7560 reviewed (changes requested):
Dans l'ensemble ca me convient.  
Cependant pas mal de changement mineurs sont à effectuer.

> - ``event`` serais à mettre en français plutôt donc ``événements``.

Pour plus de détails, regarder les notations que j'ai put faire.

ShockedPlot7560 commented:
Par ailleurs, une relecture générale est à effectuer, de nombreux accents, fautes se sont dissimulés.

ShockedPlot7560 committed:
fix exit status

QuentinGrnr committed:
Correction défauts

ce commit contient les différentes révisions demandées (orthographes, utilisation de texture du pack vanilla, rename de fichier en anglais etc...)

QuentinGrnr committed:
Merge branch 'stable' of https://github.com/ClueXIII/pluto-wiki into stable

ShockedPlot7560 commented on docs/events-auto/chest-refield.md:
```diff
@@ -0,0 +1,41 @@
+---
+id: chest-refield
```
chest-refill plutôt ?

ShockedPlot7560 commented on docs/events-auto/chest-refield.md:
```diff
@@ -0,0 +1,41 @@
+
+## Où trouver ces coffres (xyz)
+### Serveur Stelite :
+**Coffre N°1** : -325 91 -339  
```
Les axes sont toujours à préciser ?
Style :
> x: 325 y: 91 z:-339

ShockedPlot7560 commented on docs/events-auto/koth.md:
```diff
@@ -0,0 +1,31 @@
+icon: "textures/items/diamond_sword.png"
+---
+___
+<img class="thumbnail-right" src="https://user-images.githubusercontent.com/109299545/182162108-820aa6b6-80a2-440d-bb29-bbd69ad2b17b.png" alt="le KOTH du stelite">
```
Alors il va falloir soit la laisser en bas, soit la reformater, le format ne va pas du tout là :/

ShockedPlot7560 commented on docs/events-auto/koth.md:
```diff
@@ -0,0 +1,31 @@
+Les heures durant lesquelles se déroulent l'évent KOTH sont indiquées dans #eventauto/planning  
+
+**Coordonnées (xyz)**  
+**Stelite** : 201 80 -292  
```
même chose pour les coordonnées

ShockedPlot7560 commented on docs/events-auto/planning.md:
```diff
@@ -0,0 +1,12 @@
+id: planning
+title: [NEW] Planning des événements automatique 
+category: events-auto
+description: Planning relatant les heures et jours auxquels se déroulent les event automatique
```
``les évènements automatique``*

ShockedPlot7560 commented on docs/events-auto/totem.md:
```diff
@@ -0,0 +1,31 @@
+title: [NEW] Les totems en warzone
+category: events-auto
+description: Soyez les premier à détruire le totem 
+icon: "https://user-images.githubusercontent.com/109299545/182046178-97bf8b14-9f27-4dc3-a8bf-0556e2cddc21.png"
```
l'image est à changer par une texture du pack

ShockedPlot7560 commented on docs/events-auto/totem.md:
```diff
@@ -0,0 +1,31 @@
+icon: "https://user-images.githubusercontent.com/109299545/182046178-97bf8b14-9f27-4dc3-a8bf-0556e2cddc21.png"
+---
+___
+<img class="thumbnail-right" src="https://user-images.githubusercontent.com/109299545/182163339-097a92d4-c679-4e26-a395-821ead737e1a.png" alt="le totem du stelite">
```
Format de l'image ne va pas convenir

ShockedPlot7560 commented on docs/events-auto/totem.md:
```diff
@@ -0,0 +1,31 @@
+Les heures durant lesquelles se déroulent l'évent totem sont indiquée dans #events-auto/planning  
+
+**Coordonnées (xyz)**  
+**Stelite** : -265 74 268  
```
Même chose poru les coordonnées

ShockedPlot7560 reviewed (changes requested):
> - Le dossier ``.idea`` est à supprimer
> - Le fichier ``php`` est à supprimer

Des fautes d'orthographes sont toujours là et des event sont toujours présents ;)

QuentinGrnr committed:
Correction défauts (2)

QuentinGrnr committed:
Merge branch 'stable' of https://github.com/ClueXIII/pluto-wiki into stable

QuentinGrnr committed:
conflict fix bug

QuentinGrnr committed:
fix bug (2)

QuentinGrnr committed:
correction fautes d'orthographes

j'ai littéralement passé chaque phrase dans un correcteur mdrr
